box/schema_test.go

```diff
@@ -0,0 +1,16 @@
+package box
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewSchema(t *testing.T) {
+	// Create a schema instance with a nil connection. This should lead to a panic later.
+	b := NewSchema(nil)
+
+	// Ensure the schema is not nil (which it shouldn't be), but this is not meaningful
+	// since we will panic when we call the schema methods with the nil connection.
+	require.NotNil(t, b)
```

Comment:
```suggestion
	require.NotNil(t, b)
	require.Nil(t, b.User())
```

Reply:
it might be not nil, cause we panics only on nil connection

Reply:
i added test with info method call panics, cause conn is nil

i think resolved